composer.json

```diff
@@ -21,19 +15,20 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0 <8.2",
+        "ext-json": "*",
+        "ext-mbstring": "*",
         "guzzlehttp/guzzle": "^7.4.1",
-        "psr/log": "^1.0",
         "laminas/laminas-cache": "^3.1",
         "laminas/laminas-cache-storage-adapter-filesystem": "^2.0",
         "laminas/laminas-cache-storage-adapter-memory": "^2.0",
-        "laminas/laminas-cache-storage-deprecated-factory": "^1.0"
+        "laminas/laminas-cache-storage-deprecated-factory": "^1.0",
+        "php": "^7.4 || ^8.0 <8.3",
+        "psr/log": "^1 | ^2 | ^3"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5"
     },
     "config": {
-        "optimize-autoloader": true,
```

Comment:
@liayn Is there a reason you dropped this?

Reply:
https://getcomposer.org/doc/articles/autoloader-optimization.md says this should not be enabled for development.